src/components/MapFacetContents.js

```diff
@@ -53,6 +62,9 @@ class MapFacetContents extends React.Component<MapFacetContentsDefaultProps, Map
   static defaultProps = {
     size: null,
     mapboxKey: '',
+    glyph: 'map-marker',
+    uri: null,
+    tooltip: 'occurrence(s)',
```

Comment:
We could update this default value to be `'{} occurrence|{} occurrences'` so it would handle the single/plural automatically.

Reply:
Done